demo/demo.sh

```diff
@@ -25,8 +25,8 @@ run () {
         echo "$title '$t'; $(cprofile ${BASE_PATH}/$@) 2>&1 | tee ${BASE_PATH}/$t.log; sleep infinity"
 }
 
-run_router="$(run router soma-subscriptions-producer)"
```

Comment:
Then the usual way to do this is to add an `entry_point` script using distutils / setuptools and using the entrypoint script your argument to `cProfile`.